src/bin/stratisd.rs

```diff
@@ -75,15 +69,21 @@ fn main() {
         .get_matches();
 
     if matches.is_present("debug") {
-        unsafe { DEBUG = true }
-    };
+        let mut builder = LogBuilder::new();
```

Comment:
builder.init().unwrap() doesn't read the RUST_LOG env variable.  So, this code would ignore RUST_LOG.